hibernate-core/src/test/java/org/hibernate/orm/test/strategyselectors/DefaultDialectSelectorTest.java

```diff
@@ -21,8 +25,6 @@ public class DefaultDialectSelectorTest {
 	@Test
 	public void verifyAllDialectNamingResolve() {
 		testDialectNamingResolution( DB2Dialect.class );
-		testDialectNamingResolution( DB2390Dialect.class );
-		testDialectNamingResolution( DB2390V8Dialect.class );
```

Comment:
Should there be a dedicated test in the `hibernate-community-dialects` module, then? I understand we don't guarantee that dialects work, but I'd expect us to guarantee that name resolution, at least, does?

Reply:
I'll move the test assertions into that module.